changelog.txt

```diff
@@ -1,3 +1,18 @@
+4.41.0
+==========
+### New Features:
+**Use AAD plugin's account selector if authority is AAD only. This overcomes the issue of console apps not being able to display the account picker and other issues with Account Picker instability. See [2289](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2289)
```

Comment:
```suggestion
**MSAL now uses the AAD plugin's account selector if authority is AAD only.** This overcomes the issue of console apps not being able to display the account picker and other issues with Account Picker instability. See [2289](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2289)
```

Reply:
You might want to write the **WAM** AAD plugin? (Without context, I cannot understand this sentence)